activesupport/lib/active_support/core_ext/module/delegation.rb

```diff
@@ -197,9 +197,34 @@ def delegate(*methods, to: nil, prefix: nil, allow_nil: nil, private: nil)
       method_name = prefix ? "#{method_prefix}#{method}" : method
       method_names << method_name.to_sym
 
+      begin
+        arity = \
+          if to.is_a?(Class)
```

Comment:
```suggestion
          if to.is_a?(Module)
```

I think we can accept modules as well.